### PR TITLE
Add JS Live Monitor.

### PR DESCRIPTION
**Pull request by HelenaEksler**

### CODE REVIEW

**Comment by amitaibu** [comment]
build ID: 21706
public token: 77c40f0079b809f297082c1be2741375fde105e8

**Comment by HelenaEksler** [comment]
@amitaibu What is the shoov backend url?

**Comment by amitaibu** [comment]
same one - https://live-shoov.pantheon.io

File: app/js/js-live-monitor.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      return;
+    }
+
+    errors.push(row.id);
+  });
+
+  var request = new XMLHttpRequest();
+
+  var data = {
+    build: buildId,
```
Comment (amitaibu) [inline app/js/js-live-monitor.js]: I see the URL field is required in the JSLM incident, so maybe pass a `/` value in the URL from the client side

create_js_live_monitor_incident___shoov

**Comment by HelenaEksler** [comment]
@amitaibu Could you review, please.

**Pull request merged**